pegasus/test/test_hoc_routes.rb

```diff
@@ -9,6 +9,7 @@ class HocRoutesTest < Minitest::Test
       $log.level = Logger::ERROR # Pegasus spams debug logging otherwise
       @mock_session = Rack::MockSession.new(MockPegasus.new, 'studio.code.org')
       @pegasus = Rack::Test::Session.new(@mock_session)
+      Gatekeeper.set('hoc_activity_writes_disabled', value: false)
```

Comment:
In test mode we should be using the memory store which does support clear; if it is using the JSONFileAdapter something is not right.